application-accelerator/configuration.hbs.md

```diff
@@ -421,6 +421,80 @@ spec:
 
 If you are using the Tanzu CLI, then add the `--secret-ref` flag to your `tanzu accelerator create` command and provide the name of the secret for that flag.
 
+## <a id="enabling-tls"></a> Enabling TLS for Accelerators Server
```

Comment:
Can we make this singular  across the file "Accelerator Server" and "Accelerator Engine"